operations/network/management_network/bican_technical_details.md

```diff
@@ -0,0 +1,281 @@
+
+Once added to CSI, names and IP's will use the standard (v1.4) data flow and end up in SLS and be available for use via both DNS and DHCP services.
+
+<a name="touchpoints:-effects-and-changes"></a>
+### Touchpoints: effects and changes
+
+- Will require installation and admininistration document changes.
+- The CHN requires a small change in CSI to add this network.
+- Will (automatically) use the DNS infrastructure from v1.4.
+- Name aliases can be added/changed/removed via the API to SLS and become available in DNS automatically.  DNS tooling for this was released in Shasta v1.4.
+
+<a name="when-naming-occurs"></a>
+### When naming occurs
+
+- Installation as part of Cray Site Initialization (CSI) data.
+- During site customizations.
+
+<a name="ability-to-change-post-install"></a>
+### Ability to change post-install
+
+- YES - Add/Remove/Change Aliases
+- NO Change FQDN and domain suffixes because of the number of touchpoints.  Chiefly Kubernetes limitations introduced at install time and sheer number of touchpoints.
+
+<a name="endpoint-addressing"></a>
+## Endpoint Addressing
+
+For the CSM-1.2 release CHN endpoints will have **IPv4 addressing only** , with IPv6 introduction in a future release.  The current limitation to system introduction of IPv6 is Kubernetes Weave as well as a vast amount of system configuration and testing required to certify IPv6 system-wide.
+
+The CHN will, by default, have a private IPv4 address block.  This is intended to be changed during installation to a **customer-supplied IPv4 address block**.
+
+<a name="touchpoints:-effects-and-changes"></a>
+### Touchpoints: effects and changes
+
+- Installer and documentation changes to support new network and pathing as part of configuration.
+- CSI for network generation and initial configuration.
+- NCN images to support additional **subnets and routing.**
+- CFS images for CN and UAN **addressing and routing.**
+- UAI to support changes to **addressing and routing.**
+- MetalLB to create new API endpoints and peer with Edge Router.
+- Arista switch pair to create new or add to existing virtual routing instance for pathing and access control.
+- HSN required for transport of application traffic so new procedures need to be developed for troubleshooting and support.
+
+<a name="when-addressing-occurs"></a>
+### When addressing occurs
+
+- Installation as part of CSI data.
+
+<a name="ability-to-change-post-install"></a>
+### Ability to change post-install
+
+- NO - There are dozens of touchpoints throughout the system.
+
+<a name="traffic-separation-and-routing"></a>
+## Traffic Separation and Routing
+
+In CSM-1.2 the Slingshot HSN will likely not be able to support VLAN separation.  Thus there will be Layer 3 separation internal to the system but co-mingled Layer 2 between the CHN IPv4 addressing and the internal HSN private IPv4 addresses.  Isolation will be within the Slingshot network as well as separated at the Edge Router.
```

Comment:
Slingshot supports VLANs as of Slingshot v1.7 (possibly 1.6 but I know it's in use on systems running 1.7)